fdbrpc/SimExternalConnection.h

```diff
@@ -41,6 +41,10 @@ class MockDNS {
 	void removeMockTCPEndpoint(const std::string& host, const std::string& service);
 	void clearMockTCPEndpoints();
 	std::vector<NetworkAddress> getTCPEndpoint(const std::string& host, const std::string& service);
+	const std::map<std::string, std::vector<NetworkAddress>> getMockDNS() const { return hostnameToAddresses; }
+	void setMockDNS(const std::map<std::string, std::vector<NetworkAddress>>& mockDNS) {
```

Comment:
Consider override the "=" operator for better readability. `Set` usually only sets a single entity.